relnotes/decorator.feature.md

```diff
@@ -0,0 +1,12 @@
+* `ocean.util.serialize.contiguous.Serializer`
```

Comment:
Isn't this release note for the previous commit?

Reply:
Ping. Not too important, but nice to have.